### DISCUSSION TITLE
To Make The Icon Fully Adaptive.

### DISCUSSION THREAD

**Discussion by blogmangit** [uazo/bromite-buildtools#101] [Show and tell]
Dear Uazo,

In order to make the icon fully adaptive and remove any white background, you don't have to create anything new but you have to change and replace some files actually two of them. It is as follows:

Inside the res directory there are five folders namely

1. midmap-hdpi-v4
2. midmap-mdpi-v4
3. midmap-xhdpi-v4 
4. midmap-xxhdpi-v4 
5. midmap-xxxhdpi-v4 

Inside all these folders there are four files each

1. app_icon.png
2. layered_app_icon.png
3. layered_app_icon_background.png 
4. shortcut_icon_shadow.png

What you have to do is 

1. remove the file layered_app_icon.png 
2. rename layered_app_icon_background.png to layered_app_icon.png 
3. Then make a copy of the file shortcut_icon_shadow.png and rename that copy to layered_app_icon_background.png 

Your do this for all the five folders and your job will be done the icon will be 100% adaptive without any white or black background.

![22_03_2023_22_59_22.jpg](https://user-images.githubusercontent.com/92579403/226989510-5eb34406-c0d5-4e4e-9e0b-0743d1c3b612.jpg)

**Comment by uazo** [comment]
I can try, but graphics are not really in my thing.
I anticipate that it is a very very very low priority for me.
